src/components/Dissolution/CareAndCustodySelect.vue

```diff
@@ -18,9 +18,8 @@
         >
           <v-radio-group
             v-model="liquidatorOrCustodian"
-            column
             class="pt-0 mt-0 care-and-custody-option-list"
-            @change="changeCareAndCustodyType()"
+            @update:modelValue="changeCareAndCustodyType()"
```

Comment:
Apparently that's the new syntax for `@change` in Vue 3.

Reply:
What event does this component emit? Surely its name must be part of line 22.

Reply:
I think this is the usual `@change` emitter that is used in v-text-fields for example. The name has been changed to the above line.

Reply:
Oh, I thought this was one of our components. The event is probably `modelValue` here.

Reply:
For reference: 
![CareAndCustodySelect](https://github.com/bcgov/business-create-ui/assets/122301442/bc9c710f-cbf3-4f8e-932a-3c64b18c3157)